### ISSUE TITLE
TypeError in tests/deprecation_switch_testnet.py

### ISSUE THREAD

**Issue by pirapira** [raiden-network/raiden-contracts#416]
Before working on #380, I want to make sure I can interact with `tests/deprecation_switch_testnet.py`.  Currently I see a TypeError.

This issue is visible only after fixing #389.

```
(env) ~/src/raiden-contracts(389) $ python raiden_contracts/tests/deprecation_switch_testnet.py  --private-key ~/.local/share/io.parity.ethereum/keys/kovan/UTC--2019-01-08T17-09-24Z--fcf0e5e3-7d74-7abc-773e-a2ccab085547 
Web3 provider is RPC connection http://127.0.0.1:8545
Enter the private key password: 
DEBUG:raiden_contracts.deploy.__main__:Deploying EndpointRegistry txHash=0x96630099849ed26d0ca61093036b4faec40053652139a48a365ef99c8f11f32b
INFO:raiden_contracts.deploy.__main__:EndpointRegistry address: None. Gas used: None
Traceback (most recent call last):
  File "raiden_contracts/tests/deprecation_switch_testnet.py", line 262, in <module>
    deprecation_test()
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "raiden_contracts/tests/deprecation_switch_testnet.py", line 71, in deprecation_test
    ) = deprecation_test_setup(deployer, token_amount)
  File "raiden_contracts/tests/deprecation_switch_testnet.py", line 103, in deprecation_test_setup
    deployed_contracts = deploy_raiden_contracts(deployer)['contracts']
  File "/home/yh/src/raiden-contracts/raiden_contracts/deploy/__main__.py", line 376, in deploy_raiden_contracts
    endpoint_registry_receipt['contractAddress'],
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/eth_utils/address.py", line 129, in to_checksum_address
    norm_address = to_normalized_address(value)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/eth_utils/address.py", line 75, in to_normalized_address
    hex_address = hexstr_if_str(to_hex, value).lower()
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/eth_utils/conversions.py", line 157, in hexstr_if_str
    return to_type(primitive, hexstr=hexstr)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/eth_utils/decorators.py", line 73, in wrapper
    _assert_one_val(*args, **kwargs)
  File "/home/yh/src/raiden/env/lib/python3.6/site-packages/eth_utils/decorators.py", line 39, in _assert_one_val
    "Instead, values were: %r, %r" % (args, kwargs)
TypeError: Exactly one of the passed values can be specified. Instead, values were: (None,), {'hexstr': None}
```

**Comment by pirapira** [comment]
This is a duplicate of #347 .

**Comment by pirapira** [comment]
This was not a duplicate. The problem was underestimated gas consumption.
```
Gas Limit:
5500000
Gas Used By Transaction:
585784 (10.65%) 
```
https://kovan.etherscan.io/tx/0x24aaa4f3416acbf349cfa58779035b792ebc71420f9238f7de32898a77a908d2

**Comment by pirapira** [comment]
The behavior on different parity command line options.  I can see this issue with
```
./target/release/parity --chain=kovan --jsonrpc-apis web3,eth,net,parity,traces,rpc,personal
```

**Comment by loredanacirstea** [comment]
Yes, seems like a duplicate at first sight.

**Comment by pirapira** [comment]
I agree.
